protocol/communities/community.go

```diff
@@ -1234,6 +1236,31 @@ func canDeleteMessageForEveryonePermissions() map[protobuf.CommunityMember_Roles
 	return roles
 }
 
+func (o *Community) GetAdminMembers() []*ecdsa.PublicKey {
+	return o.getMembersByPredicate(o.IsMemberAdmin)
+}
+
+func (o *Community) GetTokenMasterMembers() []*ecdsa.PublicKey {
+	return o.getMembersByPredicate(o.IsMemberTokenMaster)
+}
+
+func (o *Community) GetOwnerMembers() []*ecdsa.PublicKey {
+	return o.getMembersByPredicate(o.IsMemberOwner)
+}
+
+type predicate func(pk *ecdsa.PublicKey) bool
+
+func (o *Community) getMembersByPredicate(filterFn predicate) []*ecdsa.PublicKey {
+	result := make([]*ecdsa.PublicKey, 0)
+	members := o.GetMemberPubkeys()
+	for _, member := range members {
+		if filterFn(member) {
+			result = append(result, member)
+		}
+	}
+	return result
+}
```

Comment:
this is not needed anymore
I added `GetPrivilegedMembers` which will do the same

Reply:
Removed!